Entities/Object/Carousel.php

```diff
@@ -6,7 +6,7 @@
 /**
  * Carousel Entity
  */
-class Carousel extends Entities\Object
+class Carousel extends Entities\MindsObject
```

Comment:
I went back and forth on whether to move all of the classes in the `Object` namespace to `MindsObject` as well, but in the interests of keeping changes to a minimum, I decided against doing so. Nothing seems to break by leaving it that way.